README.md

```diff
@@ -634,6 +634,16 @@ and run
 npm run lint-all-the-things
 ```
 
+Many eslint errors and warnings can be automatically fixed using the `--fix` command line argument. For example,
+
+```bash
+./node_modules/.bin/lint-all-the-things --fix
+```
+
+```bash
+npm run lint-all-the-things -- --fix
```

Comment:
Two sets of `--`?

And does this work?  I think when I tried this it did not but I have done _a lot_ of stuff since then and don't remember 😄

Reply:
Yup, I've confirmed that it works :) The extra `--` is needed to pass arguments through `npm run` to the underlying script.

Reply:
Ahh, that's why it didn't work for me.  I didn't use the extra `--` which I remember you need in this situation as you describe 😄